spring-context/src/test/java/org/springframework/context/support/GenericApplicationContextTests.java

```diff
@@ -249,11 +249,12 @@ private void assertGetResourceSemantics(ResourceLoader resourceLoader, Class<? e
 		if (resourceLoader != null) {
 			context.setResourceLoader(resourceLoader);
 		}
-
+		String commonLocation = "foo";
```

Comment:
```suggestion
		String relativePathLocation = "foo";
```

Please rename the variable.